### ISSUE TITLE
Feature Request: Open "Load external" file chooser dialog at the folder of playing file

### ISSUE THREAD

**Issue by delight-aug** [celluloid-player/celluloid#673]
When I watch a movie or series, I often have to load a subtitle or audio track from somewhere inside the folder of playing video file. When I click on "Load External" it opens a file chooser dialog on it's "Recent" tab, where I can see files I've already used, e.g. tracks for last episode. It isn't very useful, and then I have to navigate to the thing from the beginning, by clicking on "Videos", then on folder with name of the thing to watch, then "ENG Sub" folder, then choose a proper file. Also, for other people, the path could be way longer. It would be nice if it opened a file chooser dialog inside the folder of playing video, so I can just go to "ENG Sub" and choose needed file.

**Comment by Iss-in** [comment]
yes pls, this is extremely important, by default, the dilogue box is opened in recent folder, not the media folder. its a nightmare to go all the way to media file again from the home
